### ISSUE TITLE
yarn create comit-app foo

### ISSUE THREAD

**Issue by D4nte** [comit-network/create-comit-app#113]
# Problem

Currently, to create a new project the user needs to do `yarn create comit-app new foo`, this is strange as `new` and `create` convey the same meaning.

Also, users may be used to react-app that is used as follow: `yarn create react-app foo`.

# Goal

Avoid redundant commands and make it similar to react-app as it is familiar.

# Recommendation

the `bin` needs to be another js/ts script that passes `new` to the create-comit-app real binary.

**Comment by D4nte** [comment]
According to https://yarnpkg.com/lang/en/docs/cli/create/

> `yarn create <starter-kit-package> [<args>]`
> This command is a shorthand that helps you do two things at once:
> - Install create-<starter-kit-package> globally, or update the package to the latest version if it already exists
> - Run the executable **located in the bin field of the starter kit’s package.json**, forwarding any <args> to it


So actually, the `bin` field of the `package.json` need to be modified.
If we do that, then we will not be able to use this bin to do `start-env`.
Hence, #114 should be done first.

**Comment by D4nte** [comment]
Superseded by #114

**Comment by luckysori** [comment]
Reopened because when doing #114 I followed the original recommendation which doesn't actually work :face_with_head_bandage:

**Comment by D4nte** [comment]
> Reopened because when doing #114 I followed the original recommendation which doesn't actually work 🤕

We knew the original recommendation actually did not work. If the rest of #114 is done except for the `new` then we need a new idea for the `new`. Most likely, have a bin which is a TS file proxying create-comit-app.

**Comment by bonomat** [comment]
@D4nte : is this fixed already?

**Comment by D4nte** [comment]
Done with #280
